techniques/system/server-roles/1.0/password-check.st

```diff
@@ -55,6 +55,15 @@ bundle agent root_password_check_disclaimer
 
 }
 
+bundle agent root_password_security
+{
+  files:
+    # ensure password access is forbidden
+    root_server::
+      "${managed_files}"
```

Comment:
it also needs at least any .rpmnew files that might have been created by package (and probably dpkg-new ?)
and also rudder-users.xml, rudder-slapd.conf and rudder_networks.*